sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -58,6 +58,7 @@ export interface CertificatePolicy {
     issuerParameters?: IssuerParameters;
     keyProperties?: KeyProperties;
     lifetimeActions?: LifetimeAction[];
+    // Warning: (ae-forgotten-export) The symbol "SecretProperties" needs to be exported by the entry point index.d.ts
```

Comment:
The PR description says that `SecretProperties` is not needed. Then why is it part of `CertificatePolicy`?

Reply:
No idea! I'll check

Reply:
It should be fine now! I brought SecretProperties back again.